src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -371,13 +371,15 @@ extern "C" {
 
 /**
  * Maximum allowed ICE configuration user name length
+ * https://docs.aws.amazon.com/zh_tw/kinesisvideostreams/latest/dg/API_AWSAcuitySignalingService_GetIceServerConfig.html#API_AWSAcuitySignalingService_GetIceServerConfig_RequestSyntax
```

Comment:
minor: Could you remove `zh_tw` from the link? This will automatically detect your locale.

From:
https://docs.aws.amazon.com/zh_tw/kinesisvideostreams/latest/dg/API_AWSAcuitySignalingService_GetIceServerConfig.html#API_AWSAcuitySignalingService_GetIceServerConfig_RequestSyntax

To:
https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_AWSAcuitySignalingService_GetIceServerConfig.html#API_AWSAcuitySignalingService_GetIceServerConfig_RequestSyntax

Reply:
+1

Reply:
Indeed. corrected it. thanks.